client/dist/styles.css

```diff
@@ -221,26 +222,100 @@ body {
 
 /* Bishal Related Products & Outfit Creation */
 
-/* Cards CSS */
-.carousel-card-container {
+/* Related Products Container */
+
+.related-product-list-container, .outfit-list-container {
+  max-width: fit-content;
+  margin: 30px auto 15px auto;
+}
+
+
+/* Carousel CSS */
+.carousel-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.carousel-wrapper {
+  display: flex;
+  width: 100%;
+  position: relative;
+}
+
+.carousel-content-wrapper {
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+}
+
+.carousel-content {
+  display: flex;
+  gap: 10px 10px;
+  transition: all 250ms linear;
+
+}
+
+
+/* .carousel-content > * {
+  width: 100%;
+  flex-shrink: 0;
+  flex-grow: 1;
+} */
+
+
+.left-arrow, .right-arrow {
+  position:absolute;
+  z-index: 9000;
```

Comment:
Do the arrows appear on the page at all times? If so, could you please dial the z-index down a little so they don't bleed through my review modals? :grimacing:

Reply:
changed to 1